packages/react/src/components/TextArea/TextArea.js

```diff
@@ -76,17 +139,29 @@ const TextArea = React.forwardRef(function TextArea(
     [`${prefix}--text-area--invalid`]: invalid,
   });
 
-  const input = (
+  const inputField = (
     <textarea
       {...other}
```

Comment:
Do we need to worry about `other.defaultValue` being spread to this?

Reply:
no I don't think so, since it's destructured